projects/igniteui-angular/src/lib/grids/grid/grid-row.component.ts

```diff
@@ -54,6 +54,19 @@ export class IgxGridRowComponent extends IgxRowDirective<IgxGridComponent> {
         };
     }
 
+    get mrlRightPinnedOffset(): string {
```

Comment:
Didn't we already had that method somewhere else?
Is it different and can we use this one when we need the pinned offset?